lib/DynamoRequest.js

```diff
@@ -24,10 +24,7 @@ DynamoRequest.prototype.setTable = function (prefix, table) {
 
 DynamoRequest.prototype.setItem = function (item) {
   if (item) {
-    this.data.Item = {}
-    for (var key in item) {
-      this.data.Item[key] = typeUtil.valueToObject(item[key])
-    }
+    this.data.Item = typeUtil.packObjectOrArray(item)
```

Comment:
I know it's relatively unrelated to this change, but it would really help me understand what's going on if you could flesh out the docs in `typeUtil`. For example, `typeUtil.valueToObject` has no description and no return type specified.